vcstool/commands/command.py

```diff
@@ -83,7 +83,7 @@ def existing_dir(path):
 
 
 def simple_main(parser, command_class, args=None):
-    add_common_arguments(parser)
+    add_common_arguments(parser, path_nargs='?')
```

Comment:
This is not the right way to address the regression. Before `simple_main` was using the default value `single_path=False` and the semantic of the default hasn't changed: `path_nargs='*'`.